content/en/docs/Reference/image_matrix.md

```diff
@@ -60,14 +60,21 @@ The artifacts can be found in the `build` directory.
 
 ### Building core and standard generic images
 
-Unfortunately we don't have the resources and capacity to build every possible artifact in our matrix. Thankfully, you can still build those images manually on your local machine, all you need is [git](https://git-scm.com/), [docker](https://www.docker.com/) and [earthly](https://earthly.dev/). Here's an example how to build an Almalinux ARM generic image
+Unfortunately we don't have the resources and capacity to build every possible artifact in our matrix. Thankfully, you can still build those images manually on your local machine, all you need is [git](https://git-scm.com/), [docker](https://www.docker.com/). Here's an example how to build an Almalinux ARM RPI4 container image.
 
 ```bash
 git checkout https://github.com/kairos-io/kairos.git
 cd kairos
-earthly +all-arm-generic --FAMILY=@family --FLAVOR=@flavor --FLAVOR_RELEASE=@flavorRelease --BASE_IMAGE=@baseImage --VARIANT=core
+docker build --platform arm64 --build-arg BASE_IMG=almalinux:9 --build-arg MODEL=rpi4 --build-arg VERSION=1.0.0 -f images/Dockerfile -t mycustomimage:1.0.0 .
```

Comment:
Is `--platform linux/arm64` the same as `--platform arm64`? I've always used the first one.

Reply:
if no OS is given it defaults to the current. So yeah usually the same unless you run from mac in which case it will be darwin/arm64 lol. Let me change this